### PR TITLE
Styled Components Refactor

### PR DESCRIPTION
**Pull request by jdetle**
This is a big refactor, is there a way I can use now to spin up a preview?

### CODE REVIEW

**Comment by jdetle** [comment]
https://nteract-site-pgljuisokb.now.sh <- please use this to review the site.

**Comment by jdetle** [comment]
Hi @kelleyblackmore, I can't see which section you're referring to. This is a big refactor so I'm sure there are errors and I appreciate any help you can offer.

File: pages/atom.js [hunk 1]
```diff
@@ -1,143 +1,74 @@
 // @flow
```
Comment (kelleyblackmore) [inline pages/atom.js]: @jdetle these examples

Comment (jdetle) [reply pages/atom.js]: Oops! Must have accidentally deleted that in a rebase. Good catch! Thanks.

**Comment by jdetle** [comment]
https://nteract-site-teapsakxrm.now.sh/atom <- what do you think @kelleyblackmore

**Comment by kelleyblackmore** [comment]
> https://nteract-site-teapsakxrm.now.sh/atom <- what do you think @kelleyblackmore

Also with that recent deploy It doesn't display background svg?
I'm looking to see if it's just me or a change you just made

**Comment by kelleyblackmore** [comment]
nteract_pup

**Comment by jdetle** [comment]
z-index bug, thanks!

**Comment by jdetle** [comment]
https://nteract-site-gidfocloqh.now.sh

**Comment by willingc** [comment]
The color seems a bit off on the background. Was that a conscious design decision? 

One thing that concerns me a bit is the contrast between the navbar colors for accessibility. We should resolve this, but it doesn't need to be done in this refactor PR.

Thanks @jdetle 👍

**Comment by jdetle** [comment]
It was, but I can go ahead and change it back. I just figured it was nice to change something like that in a refresh. Are there good tools to audit accessibility for these sort of things @willingc?

File: package.json [hunk 2]
```diff
@@ -1,56 +1,65 @@
...
-    "{lib,spec}/**/*.{js,scss}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nteract/nteract.io.git"
-  },
+  "version": "1.0.0",
```
Comment (willingc) [inline package.json]: Do we want to bump this in the refactor?

File: package.json [hunk 3]
```diff
@@ -1,56 +1,65 @@
...
     "babel-plugin-wrap-in-js": "^1.1.1",
+    "express": "^4.16.3",
     "fetch-github-organization": "^1.0.3",
     "glob": "^7.1.2",
     "in-view": "^0.6.1",
     "isomorphic-fetch": "^2.2.1",
     "mdi": "^2.2.43",
-    "next": "^6.0.0",
+    "mdi-react": "^3.3.0",
+    "next": "^7.0.2",
```
Comment (willingc) [inline package.json]: Yay!

File: package.json [hunk 4]
```diff
@@ -1,56 +1,65 @@
...
-    "react-dom": "^16.3.2",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-fns": "^1.4.0",
     "react-headroom": "^2.2.2",
+    "react-spring": "^5.1.7",
     "react-syntax-highlighter": "^7.0.2",
-    "sass-loader": "^7.0.1"
+    "sass-loader": "^7.0.1",
+    "styled-components": "^3.2.6",
```
Comment (willingc) [inline package.json]: Do we want to update to v4 (just released) https://www.styled-components.com/releases#v4.0.0 while we are refactoring?

**Comment by rgbkrk** [comment]
> The color seems a bit off on the background. Was that a conscious design decision?

Whoa. I see what you mean @willingc. I finally got a chance to click. That is definitely hard on my eyes to read. It's a hard color to strike contrast with.

File: package.json (hunk 4 above)
Comment (jdetle) [reply package.json]: Great! I also should get rid of the sass-loader.

File: package.json (hunk 2 above)
Comment (jdetle) [reply package.json]: No, I don't know who put that there. Thanks.

**Comment by jdetle** [comment]
I'll change to the old color! Thanks for pointing out the issue.

File: package.json (hunk 2 above)
Comment (jdetle) [reply package.json]: @willingc totally misread the question and answered incorrectly. Sorry about that. I will bump the version to 2.0 I guess? Don't really know what we want to consider this.

**Comment by rgbkrk** [comment]
I think this is close enough to call it good and move forward building in new stuff. The desktop and Atom pages are missing some CSS for putting the little mock UIs "under the fold" of the white section below the hero area:

nteract.io:

screen shot 2018-10-17 at 4 50 17 pm


This PR:

screen shot 2018-10-17 at 4 49 57 pm

(Pardon the little 1.00 on the screen there, that's a chrome extension for video speed controller)

Anywho, I say this is fit to ship. Let's go for it!

**Comment by willingc** [comment]
Thanks @jdetle

**Pull request merged**